libs/fluid-elements/virtual-scroll-container/src/lib/virtual-scroll-state.ts

```diff
@@ -0,0 +1,291 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface FluidVirtualScrollOptions {
+  itemsBatchSize?: number;
+  itemsBufferSize?: number;
+}
+
+export interface FluidVirtualScrollRenderedItemRange {
+  first: number;
+  last: number;
+}
+
+// Number of items to render each time until the container is filled up
+const VIRTUAL_SCROLL_BATCH_SIZE = 5;
+// Number of items to render in addition to the initially rendered items
+const VIRTUAL_SCROLL_BUFFER_SIZE = 3;
+
+export class VirtualScrollState {
```

Comment:
General question, do we prefix exported symbols like on the angular side (`dt`, `fluid`)?

Reply:
yep, we should